### PR TITLE
chore(assets): change app name and logo images

### PR DESCRIPTION
**Pull request by developerfred**
### What does this do?

- chante app name to zOS to zero 
- change images icons 

*Before/after*
image

#### Things to consider:
  1. How will this affect security? no 
  1. How will this affect performance? no
  1. Does this change any APIs? no

### CODE REVIEW

**Comment by ratik21** [comment]
@developerfred as requested before, could you please attach before/after screenshots here so it's easier to understand the change? Thanks.

File: public/index.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon.png" />
+    <link rel="icon" href="https://framerusercontent.com/images/q863ucuBLgl5gWqHq00yE9u1TBc.png" />
```
Comment (ratik21) [inline public/index.html:5]: This should be uploaded to cloudinary first & then update the url here

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# zOS
+# Zero
```
Comment (ratik21) [inline README.md:1]:
```suggestion
# ZERO
```

File: public/index.html (hunk 1 above)
Comment (developerfred) [reply public/index.html:5]: Ok, thanks!

**Comment by developerfred** [comment]
> @developerfred as requested before, could you please attach before/after screenshots here so it's easier to understand the change? Thanks.

right, I will do this.

File: public/index.html [hunk 3]
```diff
@@ -5,10 +5,10 @@
     <link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="zOS" />
+    <meta name="description" content="ZERO" />
     <!-- added unsafe-eval here, but we should restrict this to *just* the Olm module, rather than for all self  -->
     <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline' 'unsafe-eval';" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="apple-touch-icon" href="https://framerusercontent.com/images/q863ucuBLgl5gWqHq00yE9u1TBc.png" />
```
Comment (domw30) [inline public/index.html:11]: We shouldn't change this from `%PUBLIC_URL%/logo192.png`. 

logo192, which I can see that you have updated in this pull request, is what will be used for

```
<link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Revert this change and it should be what we want to use? Not sure why it has been changed to `https://framerusercontent.com/images/q863ucuBLgl5gWqHq00yE9u1TBc.png`? Maybe I'm missing something..

**Review by domw30 — CHANGES_REQUESTED** [review]
Nice. I have one concern. I left a comment. Thanks!

File: public/index.html (hunk 3 above)
Comment (developerfred) [reply public/index.html:11]: Thanks, @domw30 , I'll switch back to the default logo192

File: public/index.html [hunk 4]
```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon.png" />
```
Comment (domw30) [inline public/index.html:5]: Hey coding, we have now uploaded the image, Could you update this line to:

`<link rel="icon" href="https://res.cloudinary.com/fact0ry/image/upload/v1686770083/zero-assets/zos/favicon2.png" />`

thank you

Comment (developerfred) [reply public/index.html:5]: sure, I will change now.

Comment (developerfred) [reply public/index.html]: done

File: public/logo192.png
Comment (domw30) [inline public/logo192.png:1]: I've just now noticed that the png you're adding here is size 512px x 512px. This should be 192px x 192px?

Comment (developerfred) [reply public/logo192.png:1]: one second, let's resize this

Comment (developerfred) [reply public/logo192.png:1]: done @domw30

**Review by domw30 — APPROVED** [review]
Thanks @developerfred. The 192px png looks a little blurred to me, but, it will probably be fine. We can always come back and change/update if needs be. 

Feel free to merge. Nice one 👍

**Comment by developerfred** [comment]
> Thanks @developerfred. The 192px png looks a little blurred to me, but, it will probably be fine. We can always come back and change/update if needs be.
> 
> Feel free to merge. Nice one 👍

I changed the size now through figma, let's see if the image looks good

**Comment by domw30** [comment]
> > Thanks @developerfred. The 192px png looks a little blurred to me, but, it will probably be fine. We can always come back and change/update if needs be.
> > Feel free to merge. Nice one 👍
> 
> I changed the size now through figma, let's see if the image looks good before/after Screenshot 2024-05-08 at 8 03 44 AM

Nice Fred. Looks awesome

**Review by domw30 — APPROVED** [review]
Looks good to me! Nice work. Remember to `Squash and Merge`

**Pull request merged**